### PR TITLE
Update/79113 incorrect subheading style and copy on wc core profiler

### LINKED ISSUE #79113: Incorrect subheading style and copy on the Woo Core Profiler Jetpack screen
#### Steps to reproduce

1. Create a JN site using WooCommerce trunk branch
2. Go to Core Profiler flow
3. Install plugins and confirm you're redirected to Jetpack connection page

Figma: Y5pUYSJPsGEud1vknUZhi8-fi-2283_38006

#### What I expected

Subheading used on the Jetpack screen should have the same style compared to the others on Core Porfiler.

Copy:

> We'll make it quick – promise. In order to take advantage of the benefits offered by Jetpack, you'll need to create a WordPress.com account.  Already have one? Log in

#### What happened instead

It’s kind of lighter.

### PR DESCRIPTION
**Pull request by moon0326**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Closes https://github.com/Automattic/wp-calypso/issues/79113

### How to test the changes in this Pull Request:

1. Checkout this branch and build the assets.
2. Start the core profiler.
3. Confirm subheading letter-spacing is 0

### Changelog entry

-   [ ] Automatically create a changelog entry from the details below.

<details>

#### Significance

-   [ ] Patch
-   [ ] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Comment 

</details>

### CODE REVIEW

File: plugins/woocommerce/changelog/update-79113-incorrect-subheading-style-and-copy-on-wc-core-profiler [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: tweak
+
+Use -webkit-font-smoothing: antialiased for core profiler subheadings';
```
Comment (chihsuan) [inline plugins/woocommerce/changelog/update-79113-incorrect-subheading-style-and-copy-on-wc-core-profiler]:
```suggestion
Use -webkit-font-smoothing: antialiased for core profiler subheadings
```

**Review by chihsuan — COMMENTED** [review]
@moon0326 Thanks for the fix! 

I'm not sure if we want to update the subheading style in the Core. I thought we wanted to fix it on Jetpack connect page since @verofasulo identified this style bug on Jetpack connect page.


Besides, I checked the CSS styles, and I noticed that `letter-spacing` difference. Core's letter-spacing is 0.25px. Could you also update it? 🙏 




Screenshot 2023-08-02 at 13 00 19
Screenshot 2023-08-02 at 13 00 16

**Comment by verofasulo** [comment]
> Core's letter-spacing is 0.25px.

@moon0326, where do you see this? Letter spacing should be 0px both for body and heading 😅

**Comment by moon0326** [comment]
> > Core's letter-spacing is 0.25px.
> 
> @moon0326, where do you see this? Letter spacing should be 0px both for body and heading 😅

@verofasulo Subheadings have 0.25px letter spacing

0.25px:
Screen Shot 2023-08-02 at 4 20 40 PM

0px:
Screen Shot 2023-08-02 at 4 20 45 PM

**Comment by moon0326** [comment]
> @moon0326 Thanks for the fix!
> 
> I'm not sure if we want to update the subheading style in the Core. I thought we wanted to fix it on Jetpack connect page since @verofasulo identified this style bug on Jetpack connect page.
> 
> Besides, I checked the CSS styles, and I noticed that `letter-spacing` difference. Core's letter-spacing is 0.25px. Could you also update it? 🙏
> 
> Screenshot 2023-08-02 at 13 00 19 Screenshot 2023-08-02 at 13 00 16

@chihsuan  Ah..gotcha :) I'll open a new PR in wp-calypso 👍 

Updated letter-spacing in https://github.com/woocommerce/woocommerce/pull/39526/commits/2c893d0c11bc829a76bcb25d1a88d97f60701dac

**Review by chihsuan — APPROVED** [review]
Thanks @moon0326 LGTM and tested well. 👍 

I think it's okay to merge this PR with just one approval.

**Pull request merged**